apps/web/next.config.js

```diff
@@ -184,6 +184,8 @@ const nextConfig = {
     "http-cookie-agent", // Dependencies of @ewsjs/xhr
     "rest-facade",
     "superagent-proxy", // Dependencies of @tryvital/vital-node
+    "superagent",
+    "formidable",
```

Comment:
What does this package do? Can we leave a comment on why we need it?

Reply:
Upgrading to next 15 was throwing errors with the Akismet package. Adding these dependencies to the `serverExternalPackages` property solved those errors. As suggested by @hbjORbj

 https://nextjs.org/docs/app/api-reference/config/next-config-js/serverExternalPackages